Data/ApplicationDbContext.cs

```diff
@@ -11,6 +11,10 @@ public class ApplicationDbContext : IdentityDbContext
         public DbSet<Entity> Entities { get; set; }
         public DbSet<Contact> Contacts { get; set; }
         public DbSet<Branch> Branches { get; set; }
+        public DbSet<Customer> Customers { get; set; }
+        public DbSet<Product> Products { get; set; }
+        public DbSet<Purchase> Purchases { get; set; }
+        public DbSet<PurchaseItem> PurchaseItems { get; set; }
 
         public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
                 : base(options)
```

Comment:
Not registered configuration for:
 - `CustomerConfiguration`
- `PurchaseItemConfiguration`
in
```c#
        protected override void OnModelCreating(ModelBuilder modelBuilder)
        {
            modelBuilder.ApplyConfiguration(new BranchConfiguration());

            base.OnModelCreating(modelBuilder);
        }
```